charts/refinery/values.yaml

```diff
@@ -496,6 +496,8 @@ podLabels: {}
 
 podAnnotations: {}
 
+deploymentAnnotations: {}
```

Comment:
To continue the pattern used in other subsections like `service` or `ingress` we can stick with calling this `annotations`.